.github/workflows/test.yml

```diff
@@ -0,0 +1,45 @@
+name: test
+
+on: [push]
+
+jobs:
+  test:
+    name: ${{ matrix.platform }} py${{ matrix.python-version }}
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      matrix:
+        platform: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: ["3.10", "3.11"]
```

Comment:
Is this outdated?

Reply:
Yes. We have this at a high priority in #197.